aspnet/fundamentals/dependency-injection.rst

```diff
@@ -171,7 +171,7 @@ You should design your services to use dependency injection to get their collabo
 
 What if you find that your classes tend to have way too many dependencies being injected? This is generally a sign that your class is trying to do too much, and is probably violating SRP - the `Single Responsibility Principle <http://deviq.com/single-responsibility-principle/>`_. See if you can refactor the class by moving some of its responsibilities into a new class. Keep in mind that your ``Controller`` classes should be focused on UI concerns, so business rules and data access implementation details should be kept in classes appropriate to these `separate concerns <http://deviq.com/separation-of-concerns/>`_.
 
-With regard to data access specifically, you can easily inject Entity Framework ``DbContext`` types into your controllers, assuming you've configured EF in your ``Startup`` class. However, it is best to avoid depending directly on ``DbContext`` in your UI project. Instead, depend on an abstraction (like a Repository interface), and restrict knowledge of EF (or any other specific data access technology) to the implementation of this interface. This will reduce the coupling between your application and a particular data access strategy, and will make testing your application code much easier.
+If you've added EF to the services container in ``ConfigureServices``, you can inject the  ``DbContext`` into your controllers. Some develpers prefer to use a repository interface to the database rather than using the injected ``DbContext``. Using an interface to encapsulate the data access logic in one place can minimize code changes when your database changes.
```

Comment:
I like how the original paragraph highlighted this as an application of SRP by connecting with the paragraph above saying "With regards to data access specifically". Can we preserve the connection somehow? 

Related to that, "using and interface to encapsulate the data access logic in one place" can help minimize how many places you will have to go change, not directly how much change there will be.